### ISSUE TITLE
[2.0 beta 11] Seek does not work until .play().

### ISSUE THREAD

**Issue by micahlisonbee** [goldfire/howler.js#502]
I have the ability with my audio player to scrub (seek) the track position to change the playhead. This works great by setting `seek(value)`.

However, if `seek(value)` is run before the track has been played, it does not work. When the track is played it starts from the beginning and does not respect the seeking.

My hack workaround is to do this every time a new track is loaded:

```js
track.play();
track.pause();
```

However, I think it's causing other issues like multiple tracks/sounds playing over the top of each other in rare cases.

**Comment by micahlisonbee** [comment]
I have pinpointed the issue in the code, though I'm not familiar enough with the codebase to make a pull request due to lack of logic understanding.

In the `init: function()` `_ended` is set to `true` by default:
`self._ended = true;`

And then at this section in `play:` the if statement that sets the `id` never returns true for the second statement because `_ended` is `true`:

```js
        // Check if there is a single paused sound that isn't ended.
        // If there is, play that sound. If not, continue as usual.
        var num = 0;
        for (var i=0; i<self._sounds.length; i++) {
          if (self._sounds[i]._paused && !self._sounds[i]._ended) {
            num++;
            id = self._sounds[i]._id;
          }
        }
```

**Comment by goldfire** [comment]
This should now be fixed with e0222f31aa8f8d50ae20b77108d87e13af7da031.

**Comment by micahlisonbee** [comment]
@goldfire Thanks!
